package.json

```diff
@@ -3,9 +3,10 @@
   "homepage": "http://postcss.com/",
   "scripts": {
     "start": "babel-node scripts/build --server --dev --open",
-    "static": "babel-node scripts/build --static --production",
+    "static": "npm run lint && babel-node scripts/build --static --production",
```

Comment:
I don't agree, the files should be linted before we build the website, not only when we test it. This'll help us catch some errors before we deploy them.